tock/employees/migrations/0034_auto_20200529_1230.py

```diff
@@ -0,0 +1,34 @@
+# Generated by Django 2.2.12 on 2020-05-29 16:30
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.core.exceptions import ObjectDoesNotExist
+
+def backfillorgs(apps, schema_editor):
+    Employees = apps.get_model('employees','UserData')
+
+    for orgs in Employees.all():
+        try:
+            i = 1
+            org_data = Employees.objects.get(organization=null)
+            orgs.organization = org_data.organization
+            orgs.organization = i
+            orgs.save
+            i++
+        except ObjectDoesNotExist:
+            pass 
```

Comment:
Thanks @cryptofilegsa!

Let's focus on defining and refining the iteration for now. We want to iterate through employees which have no organization assigned.

The `orgs` variable assigned name in the `for` statement is a bit confusing since we're iterating through instances of the `Employee`  model not `Organization`. The usage of the `Employees` queryset is also missing a piece, this would fail with an `AttributeError`

Django provides handy filtering functionality to return a QuerySet containing only records meetings that criteria. Take a look at how they're using `.filter()` in the [second part of the Django tutorial](https://docs.djangoproject.com/en/3.0/intro/tutorial02/#playing-with-the-api)